blog/2022/12-7-first-look-at-resoto-dashboards/index.md

```diff
@@ -0,0 +1,76 @@
+
+Now, we are introducing a new way to have this visual information always available: Resoto Dashboards.
+
+You may be wondering: "But why do we need yet another way to visualize data?". With our dashboards, you don't just visualize the data. Your searches and queries will be executed periodically and the results displayed in widgets automatically, so you always look at updated information. You can organize your widgets freely in the dashboard, to keep the most important metrics always within reach. You can share these dashboards across your organization so everybody that needs this information can make use of it. And of course, you can access your dashboards from everywhere as long as you have access to your resoto installation.
+
+## Key concepts
+
+A _dashboard_ is a container where you can place different _widgets_ to show relevant information. Each _widget_ has its way to display data, and thus, its configurable properties. Available widgets so far are:
+
+- Numeric Indicators
+- Line Charts
+- Heatmaps
+- Tables
+
+Of course, you can freely position and resize widgets inside a dashboard, to organize the data in the best way possible for your application.
+
+Each widget needs one or more _data sources_ to get the data that the user wants to display. These data sources have a query that is executed periodically to keep the dashboard updated. Each widget may have some limitations on what data sources it can accept, so not all data sources are available for every widget. Available data sources so far are:
+
+- Search
+- Aggregate Search
+- Time Series
+- Two-entry Aggregate Search
+
+## Resoto Example Dashboard
+
+Once you have resoto up and running, you can access the UI from a web browser in `HTTP://<your_resoto_url>/ui`. The following home screen should appear:
+
+![ui-home](./img/ui-home.png)
+
+There is a lot to do in the UI, but for now, let's just focus on the dashboard section. Click on the dashboard button, and you will be redirected to the Dashboard Manager:
+
+![ui-dashboard-manager](./img/ui-dashboard-manager.png)
+
+### Visualizing unused volumes
+
+For a quick demonstration of how dashboards work in resoto, let's just open the _Resoto Example Dashboard_ provided with the installation, double-clicking on that option in the list.
+
+This dashboard has pre-configured a lot of widgets with relevant information about your cloud infrastructure.
+
+![ui-dashboard-manager](./img/ui-example-dashboard.png)
+
+Let's take a quick look at some of the elements being displayed here. Look for the table widget titled "Unused Volumes older than 1 week". There you have a complete list, as the title suggests, of the available volumes older than 1 week. This information is really useful if you want to know what resources you may mark for cleanup, and is very easily retrieved with resoto and displayed in a dashboard.
```

Comment:
The UI screenshot here (ui-example-dashboard.png) does not show a widget with the title `Unused Volumes older than 1 week` 
I can only see `Instances Total`, `InstanceCores`, `Instance Memory`, `Instance Types by Account` and `Instances by Region`

Reply:
yeah, that widget didnt enter in the screenshot, you have to scroll down. Either I change the screenshot (but this would mean the user probably wont have the same view when opening the dashboard), or i Just write `scroll down until you find the widget titled "Unused volumes older than 1 week"`

Reply:
Oh, I see - this makes sense when you assume that people are looking at their own dashboard, not just reading through the blogpost only. Hm... I'm not sure what the best solution would be 🤔 @TheCatLady usually has good ideas though :)